### PR TITLE
Use singletons for all fallback components

### PR DESCRIPTION
**Pull request by bjorncs**
I confirm that this contribution is made under the terms of the license found in the root directory of this repository's source tree and that I have the authority necessary to make this contribution on behalf of its copyright owner.

### CODE REVIEW

**Review by baldersheim — APPROVED** [review]
Perhaps @hmusum @jonmv @gv or @bratseth Wants to have a look too.

File: jdisc_http_service/src/main/java/com/yahoo/container/logging/AccessLog.java [hunk 1]
```diff
@@ -13,6 +13,8 @@
  */
 public class AccessLog implements RequestLog {
 
+    public static final AccessLog VOID_INSTANCE = new AccessLog(new ComponentRegistry<>());
```
Comment (bratseth) [inline jdisc_http_service/src/main/java/com/yahoo/container/logging/AccessLog.java:16]: You could call this "none" ...

**Comment by bratseth** [comment]
LGTM

**Review by hmusum — APPROVED** [review]

**Review by gjoranv — COMMENTED** [review]
👍

**Pull request merged**